cmd/crictl/main.go

```diff
@@ -196,12 +203,12 @@ func main() {
 			Name:    "timeout",
 			Aliases: []string{"t"},
 			Value:   defaultTimeout,
-			Usage:   "Timeout of connecting to the server",
+			Usage:   "Timeout of connecting to the server (default: 2s)",
```

Comment:
```suggestion
			Usage:   fmt.Sprintf("Timeout of connecting to the server (default: %ds)", defaultTimeout),
```

Reply:
Thanks for the suggestion. Seemed to miss this previously but it is not needed as CLI lib prints the default.